index.js

```diff
@@ -5,7 +5,19 @@ module.exports = function (filename, opts) {
   opts = { ...opts }
 
   if (!opts.transform) {
-    opts.transform = require('./dist/babel')
+    switch (opts.transformPreset) {
```

Comment:
We may also use a map object from name to callback. This way later on can do more refactors and support 3rd party presets

Reply:
PS: It is currently not exactly preset but just transform impl. Does it makes sense having two options?

Reply:
Okay, let's revert to one.